open-company/open-company-web
# Reply button

### Merge after #1135 

Card: https://trello.com/c/4cfoFFH0
Abstract: https://share.goabstract.com/925a4d5e-5052-4ecf-acac-293fce3e7edb?mode=design&sha=e4915876d67a9e40c6ab7a8826acee00a7df21e2

Adds a Reply button after the last comment of each thread on mobile only.

To test:
- on mobile
- open a post w/o comments
- no reply button?
- add a comment
- Reply button below the comment?
- reply to your comment
- Reply button only after the reply?
- add another root comment
- 2 Reply button: one under the single root comment and one after the reply?
- reply to your previous reply
- still 2 Reply?
- thread with replies has Reply after the last one?
- open the same post on desktop
- don't see a Reply button?
